test - FAIL: front end routes - #https get /legal/privacy 404

FAIL: front end routes - #https get http://127.0.0.1:3030/legal/privacy (746ms)
AssertionError: expected 404 to equal 200

Seeing this locally and on Travis, cannot explain why

```
PASS: front end routes - #http get http://127.0.0.1:3080/cookies_disabled (8ms)
PASS: front end routes - #https get http://127.0.0.1:3030/force_auth (7ms)
PASS: front end routes - #http get http://127.0.0.1:3080/force_auth (9ms)
PASS: front end routes - #https get http://127.0.0.1:3030/legal (6ms)
PASS: front end routes - #http get http://127.0.0.1:3080/legal (7ms)
FAIL: front end routes - #https get http://127.0.0.1:3030/legal/privacy (746ms)
AssertionError: expected 404 to equal 200
  at Function.assert.equal  <node_modules/intern/browser_modules/chai/chai.js:2277:10>
  at <tests/server/routes.js:285:18>
  at process._tickCallback  <node.js:385:9>
PASS: front end routes - #http get http://127.0.0.1:3080/legal/privacy (8ms)
PASS: front end routes - #https get http://127.0.0.1:3030/legal/terms (430ms)
PASS: front end routes - #http get http://127.0.0.1:3080/legal/terms (9ms)
PASS: front end routes - #https get http://127.0.0.1:3030/metrics-errors (4ms)
PASS: front end routes - #http get http://127.0.0.1:3080/metrics-errors (4ms)
PASS: front end routes - #https get http://127.0.0.1:3030/oauth (13ms)
PASS: front end routes - #http get http://127.0.0.1:3080/oauth (7ms)
PASS: front end routes - #https get http://127.0.0.1:3030/oauth/force_auth (7ms)
PASS: front end routes - #http get http://127.0.0.1:3080/oauth/force_auth (5ms)
PASS: front end routes - #https get http://127.0.0.1:3030/oauth/signin (6ms)
PASS: front end routes - #http get http://127.0.0.1:3080/oauth/signin (6ms)
PASS: front end routes - #https get http://127.0.0.1:3030/oauth/signup (4ms)
PASS: front end routes - #http get http://127.0.0.1:3080/oauth/signup (6ms)
PASS: front end routes - #https get http://127.0.0.1:3030/report_signin (6ms)
PASS: front end routes - #http get http://127.0.0.1:3080/report_signin (6ms)
PASS: front end routes - #https get http://127.0.0.1:3030/reset_password (8ms)
PASS: front end routes - #http get http://127.0.0.1:3080/reset_password (5ms)
PASS: front end routes - #https get http://127.0.0.1:3030/reset_password_confirmed (5ms)
PASS: front end routes - #http get http://127.0.0.1:3080/reset_password_confirmed (6ms)
PASS: front end routes - #https get http://127.0.0.1:3030/reset_password_verified (5ms)
PASS: front end routes - #http get http://127.0.0.1:3080/reset_password_verified (9ms)
PASS: front end routes - #https get http://127.0.0.1:3030/settings (4ms)
PASS: front end routes - #http get http://127.0.0.1:3080/settings (6ms)
```

```
http://127.0.0.1:3030/bower_components/normalize-css/normalize.css 200
http://127.0.0.1:3030/styles/fontello.css 200
http://127.0.0.1:3030/styles/fonts/en.css 200
http://127.0.0.1:3030/styles/main.css 200
https://www.mozilla.org/privacy/ 200
https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral 200
https://support.mozilla.org/kb/cookies-information-websites-store-on-your-computer 404
FAIL: front end routes - #https get http://127.0.0.1:3030/legal/privacy (782ms)
AssertionError: expected 404 to equal 200
  at Function.assert.equal  <node_modules/intern/browser_modules/chai/chai.js:2277:10>
  at <tests/server/routes.js:269:18>
  at process._tickCallback  <node.js:385:9>
```

--- a/tests/server/routes.js
+++ b/tests/server/routes.js
@@ -282,6 +282,11 @@ define([
 
       var promise = makeRequest(url, requestOptions)
         .then(function (res) {
+          if (/support.mozilla.org/.test(url)) {
+            // Do not check support.mozilla.org URLs. Issue #4712
+            // In February 2017 SUMO links started returning 404s to non-browser redirect requests
+            return;
+          }
           assert.equal(res.statusCode, 200);
 
           var headers = res.headers;

